gui/syncthing/core/eventService.js

```diff
@@ -18,14 +18,43 @@ angular.module('syncthing.core')
                 return;
             }
             $rootScope.$broadcast(self.ONLINE);
-
+            //GREEN
+            $("link[rel='shortcut icon']").attr("href","assets/img/favicon_idle.png")
+            
             if (lastID > 0) {   // not emit events from first response
+                var green_counter=0;
+                var red_counter=0;
```

Comment:
redCounter rather than red_counter, to make it consistent.
Or just red and green.